### PR TITLE
343 add auto camera detection

### LINKED ISSUE #343: Add autodetection funcitonality to camera connect module
Refers to #317 
So far, the user has to adjust environment variables to use either an Allied Vision or a Basler camera. The user friendliness could be improved significantly if the the module would adjust automatically to the connected camera.

### PR DESCRIPTION
**Pull request by JonaVo**
Fixes #343
Cti files in docker container are automatically detected. User does not need to specify the camera's manufacturer anymore.

### CODE REVIEW

File: deployment/cameraconnect/docker-compose.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ services:
     container_name: cameraconnect_container
```
Comment (JeremyTheocharis) [inline deployment/cameraconnect/docker-compose.yml:9]: Is this change on purpose?

File: cameraconnect/example_env [hunk 2]
```diff
@@ -0,0 +1,77 @@
+### ------------- GLOBAL SETTINGS -------------
```
Comment (JeremyTheocharis) [inline cameraconnect/example_env:1]: is this change on purpose?

File: .gitignore [hunk 3]
```diff
@@ -30,7 +30,6 @@ golang/test/mqtt-bridge/data/*
 
 /.idea/
```
Comment (JeremyTheocharis) [inline .gitignore:32]: is this change on purpose?

Comment (JonaVo) [reply .gitignore:32]: no

File: cameraconnect/example_env (hunk 2 above)
Comment (JonaVo) [reply cameraconnect/example_env:1]: yes

File: deployment/cameraconnect/docker-compose.yml (hunk 1 above)
Comment (JonaVo) [reply deployment/cameraconnect/docker-compose.yml:9]: yes

**Review by JeremyTheocharis — APPROVED** [review]
